app/views/devise/sessions/new.html.haml

```diff
@@ -3,15 +3,12 @@
 
 = render partial: 'layouts/header', locals: {page_heading: t('.page_heading')}
 
-= simple_form_for(resource, as: resource_name, url: session_path(resource_name), html: { autocomplete: 'off' }) do |f|
-  .form-group
-    = f.label :email, class: "form-label-bold"
-    = f.input_field :email, autofocus: true, class: "form-control"
-  .form-group
-    = f.label :password, class: "form-label-bold"
-    = f.input_field :password, autocomplete: 'off', class: "form-control"
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    = form_for(resource, as: resource_name, url: session_path(resource_name), html: { autocomplete: 'off' }, builder: GOVUKDesignSystemFormBuilder::FormBuilder) do |f|
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3698/files#r565935858)